.github/workflows/portal-loop.yml

```diff
@@ -1,13 +1,15 @@
 name: portal-loop
 
 on:
+  pull_request: {}
```

Comment:
Can we make sure this only runs for `master` pushes? Or do you think it makes more sense on each PR, given we can always create a tag (release) from a random branch

Reply:
Since portal-loop self deploy on every master commit, it's better to know if it's gonna break before merging something :)

Reply:
Yes, but for PRs on the master branch

Reply:
```suggestion
  pull_request:
      branches:
          - master
```